src/js/disclosure/disclosure-view.js

```diff
@@ -8,7 +8,7 @@
  *
  * @module DisclosureView
  */
-;(function(define) {
+(function(define) {
```

Comment:
Yeah, I'm not sure what rule is doing this, but it's not allowed. Couple of things we can do:

- allow it explicitly in the eslint-config-edx
- allow it with an override only in places we're already doing this (edx-platform, edx-ui-toolkit)
- make sure it's not necessary any of the places we do it, and remove them all
- make the r.js optimizer or webpack do this when it concatenates them, and remove them all

Personally I'd like to get rid of them all, as I understand it it's a practice that only has a benefit when you're already doing something bad and we should just be fixing the bad code instead of allowing it to limp along :)